### ISSUE TITLE
Display localized decimal values

### ISSUE THREAD

**Issue by gamosoft** [mistic100/jQuery-QueryBuilder#713]
Hey there, sorry to bother again, I'm trying to read/write decimal values and with the default culture (English) it works fine as it displays numbers such as 12.21

When reading a saved query from backend, if I read it in another cuture (Spanish) the decimal separator is "," so I'd get a string like "12,21" which I'd like to show in the input field, however validation fails in the setter (because it's validating in English).

Is there a way to localize this easily?

Thanks again mate. :-)

**Comment by mistic100** [comment]
If you use numeric inputs, your question is irrelevant : a numeral is always represented with a dot, but the browser might display a comma in the input, you do not control that.

If you use string inputs : I have no solution for you, adapt your configuration depending of the locale of your application.

**Comment by mistic100** [comment]
In any case you should NEVER store a decimal as a string, store it as an actual decimal and format it for display only (or let the browser do it for you when using inputs).

You will go crazy if you want to compare values and handle every cases (what about hundreds separator ? space or dot or nothing...)

**Comment by gamosoft** [comment]
Absolutely, values are stored as invariant in the backend and transformed only when sent back to the user using their culture. I fixed the issue by handling double values with html **text** input fields instead of **number**. Part of the problem is that also the culture is NOT read from the browser's settings of the user, but rather some property in the system we're working on, so we can't rely on the correct decimal separator representation.
Apparently support is not very solid, or at least couldn't find an easy way to set the language in the inputs (https://www.ctrl.blog/entry/html5-input-number-localization), so I went the long way with all control myself.
Thanks again for your feedback mate! :-)
